src/sentry/api/endpoints/organization_code_mapping_details.py

```diff
@@ -28,6 +28,23 @@ def convert_args(self, request, organization_slug, config_id, *args, **kwargs):
 
         return (args, kwargs)
 
+    def _get_codeowner_contents(self, config):
```

Comment:
@MeredithAnya this func looks like a duplicate of `src/sentry/api/endpoints/organization_code_mapping_codeowners.py` line 27.